controllers/managedocs_controller.go

```diff
@@ -651,16 +659,18 @@ func (r *ManagedOCSReconciler) updateStorageClusterFromAddonParamsSecret(sc *ocs
 		}
 	}
 
+	desired := templates.StorageClusterTemplate.DeepCopy()
```

Comment:
Can we please call this `cs` instead of `desired` inside the function?
This will minimize the changes and in my eyes more meaningful in the cotext if this function

Reply:
You mean `sc`, right?
Storage cluster

Reply:
yes, sorry